sympy/physics/units/dimensions.py

```diff
@@ -572,3 +572,87 @@ def is_consistent(self):
         # dimensions
         # in vector language: the set of vectors do not form a basis
         return self.inv_can_transf_matrix.is_square
+
+    def buckingham_pi_theorem(self, list_of_derived_quantities):
+        """
```

Comment:
You can put an `r` before the string start in order to require only one backslash `\` in the string, i.e. start the string with `r""" ... `